src/dtos/login-dto.ts

```diff
@@ -0,0 +1,6 @@
+import * as yup from "yup";
+
+export default yup.object().shape({
+  email: yup.string().email().required(),
+  password: yup.string().required(),
+});
```

Comment:
How are the errors shown when validation fail? I think you can add messages like:
`email: yup.string().email('This field should be an email').required('Email is a required field'),`

and then you can probably return the validation errors with those messages